packages/app-store/_components/AppCategoryNavigation.tsx

```diff
@@ -31,7 +31,7 @@ const AppCategoryNavigation = ({
           tabs={appCategories}
           sticky
           linkProps={{ shallow: true }}
-          itemClassname={classNames(fromAdmin && "w-60")}
+          itemClassname={classNames(fromAdmin && "!w-48")}
```

Comment:
what does the `!` do here?

Reply:
Makes it `!important` in order to overwrite any other similar CSS.